ronn/geopm_agent_frequency_map.7.ronn

```diff
@@ -124,10 +126,8 @@ raised.
 
 
 ## REPORT EXTENSIONS
-The assigned frequency for each region is added to each region
-including those that were not in the policy map (which will show the
-policy default CPU frequency).
-
+The assigned frequency is added to each region's section of the report
+if that region was explicitly assigned a frequency in the policy map.
```

Comment:
Why remove the feature of having frequencies for all regions show up in the report, including those not in the policy map?

Reply:
The problem is that the Agent interface is kind of not awesome.  The agent is not given a list of all regions to report on, it has to observe them in a sample to know about them.  For this reason, we will not be able to provide the keys for the map returned by the Agent::report_region() api:  https://github.com/geopm/geopm/blob/dev/src/Agent.hpp#L121 for short running regions. In at least the case of a short running region not in the policy map, the agent is unable to meet the requirement.  So if it isn't reliably there, I said why put there for the unmapped regions.

Reply:
Please make it so that if any region runs long enough to get the default frequency from the policy, that the default frequency shows up in the individual region's section in the report (not just up in the top policy section).